roles/webserver/templates/nginx_unicorn.j2

```diff
@@ -1,5 +1,5 @@
 upstream unicorn {
-  server unix:{{ unicorn_sock }} fail_timeout=0;
+  server unix:{{ unicorn_sock }} fail_timeout=1;
```

Comment:
I didn't document doing it and it seems like it should either be zero or a number of seconds.